lib/cuda/cudaCorrelator.hpp

```diff
@@ -41,13 +45,16 @@ class cudaCorrelator : public cudaCommand {
     cudaCorrelator(kotekan::Config& config, const std::string& unique_name,
                    kotekan::bufferContainer& host_buffers, cudaDeviceInterface& device, int inst);
     ~cudaCorrelator();
+    int wait_on_precondition() override;
     cudaEvent_t execute(cudaPipelineState& pipestate,
                         const std::vector<cudaEvent_t>& pre_events) override;
+    void finalize_frame() override;
 
 protected:
 private:
     // Common configuration values (which do not change in a run)
     /// Number of elements on the telescope (e.g. 2048 - CHIME, 256 - Pathfinder).
```

Comment:
128 for the pathfinder

Reply:
I think that was an old comment about the CHIME pathfinder!  Fixed!